packages/office-ui-fabric-react/src/components/Pivot/Pivot.test.tsx

```diff
@@ -22,6 +22,23 @@ describe('Pivot', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('can be focused', () => {
+    const pivotRef = React.createRef<IPivot>();
+
+    mount(
+      <Pivot componentRef={pivotRef}>
+        <PivotItem headerText="Link 1" headerButtonProps={{ 'data-is-visible': true }} />
```

Comment:
Why does 'data-is-visible' need to be set here?

Reply:
When we say `focus()`, the `FocusZone.focus` method is called, which internally uses visibility detection to determine which item should be focused. Setting it here lets FocusZone know that this thing is visible, so that it doesn't skip over thinking it can't be focused.